src/backend/etc/nginx.conf

```diff
@@ -41,16 +36,6 @@ http {
 
     keepalive_timeout 60;
 
-    location / {
-      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-      proxy_set_header X-Forwarded-Proto $scheme;
-      proxy_set_header Host $http_host;
-      # we don't want nginx trying to do something clever with
-      # redirects, we set the Host: header above already.
-      proxy_redirect off;
-      proxy_pass http://flask_server;
-    }
-
     location /v2/ {
```

Comment:
Hrm, any issues with updating this to be `location /` and having fastapi handle all requests?

Reply:
Will try that